packages/appstore/src/assets/svgs/banner/banner_image_types.tsx

```diff
@@ -0,0 +1,5 @@
+export type BannerImageProps<T> = {
```

Comment:
if its only used in one file, do we need to keep it separate, export it, and then import it?

Reply:
Same cause there's banner in wallets project also it's better to do it this way so that later on they can reuse the types